Dockerfile

```
@@ -6,4 +6,7 @@ RUN apk --no-cache add bash bind-tools && \
 
 COPY ddns-route53 /usr/local/bin/ddns-route53
 
+ENV AWS_ACCESS_KEY_ID ACCESSKEYID
+ENV AWS_SECRET_ACCESS_KEY SECRETACCESSKEY
```

Wouldn't this just set the environment variables to the literal values `ACCESSKEYID` and `SECRETACCESSKEY` respectively?

If we did something like this then one could do `docker build --build-arg AWS_SECRET_KEY_ID=<value> --build-arg AWS_ACCESS_KEY_ID=<value> <other-arguments>`:
```suggestion
ARG AWS_ACCESS_KEY_ID
ENV AWS_ACCESS_KEY_ID $AWS_ACCESS_KEY_ID
ARG AWS_SECRET_ACCESS_KEY
ENV AWS_SECRET_ACCESS_KEY $AWS_SECRET_ACCESS_KEY
```

Although personally I'd set the variables in the environment and run it as:
```
$ docker run --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY <img>
```
As it should then pick up the values from the environment rather than embedding it in the image.

---

Yes, it looks much more elegant that way, thanks!